common_sleep.Rmd

```diff
@@ -200,224 +230,271 @@ lattice::xyplot(Reaction ~ Days|Sx,
 
 ## Between subject variability
-```{r histograms, fig.align="center"}
-## Linear regression per participant of Reaction on Days
-
-## Coefficients
-lin.reg.coef  <- by(sleep, sleep$Subject,
-                   function(data) coef(lm(Reaction ~ Days, data = data)))
-lin.reg.coef1 <- unlist(lin.reg.coef)
-names(lin.reg.coef1) <- NULL
-lin.reg.coef2 <- matrix(lin.reg.coef1,
-                        length(lin.reg.coef1)/2,
-                        2,
-                        byrow = TRUE)
-
-## R squared
-lin.reg.r.squared  <- by(sleep, sleep$Subject,
-                        function(data) summary(lm(Reaction ~ Days, data = data))$r.squared )
-lin.reg.r.squared1 <- as.vector(unlist(lin.reg.r.squared))
-
-## Histograms
-par(mfrow = c(3,1))
-hist(lin.reg.coef2[,1],
-     xlab = "Intercept",
-     col  = "lightblue",
-     main = "Histogram of individual intercepts")
-hist(lin.reg.coef2[,2],
-     xlab = "Slope",
-     col  = "lightblue",
-     main = "Histogram of individual slopes")
-hist(lin.reg.r.squared1,
-     xlab = "R squared",
-     col  = "lightblue",
-     main = "Histogram of individual R squared")
-
-#TODO? change to ggplot?
+```{r fig.height = 2.5, fig.width = 7, fig.align="center"}
+hist_intercept <- ggplot(sleep_coefs) +
+  geom_histogram(aes(Intercept),
+                 bins = 6,
+                 fill = "darkcyan")
+hist_slope <- ggplot(sleep_coefs) +
+  geom_histogram(aes(Slope),
+                 bins = 6,
+                 fill = "darkcyan")
+hist_rsquared <- ggplot(sleep_coefs) +
+  geom_histogram(aes(R_squared),
+                 bins = 6,
+                 fill = "darkcyan")
+
+cowplot::plot_grid(hist_intercept, hist_slope, hist_rsquared, 
+                   nrow = 1,
+                   labels = "AUTO")
 ```
 
-The individual intercepts shown in the first histogram correspond to the initial reaction time and are non normally distributed. Given the small data set, this is not surprising as it shows a variety of their initial reaction time. However, if this data came from a large dataset, it would be surprising that even the initial data points are not normally distributed and could suggest a wrong data sample compared to the population. 
 
-Looking at the histogram of individual slopes, we see a normal distribution. As seen on the previous graph showcasing the individual linear regressions, very little slopes are negative. This shows again that reaction time increases by days of sleep deprivation. 
+The individual intercepts shown in the first histogram (A) correspond to the initial reaction time at day zero and are non normally distributed. Given the small data set, this is not surprising as it shows a variety of the initial reaction time. However, if this data came from a large dataset, it would be surprising that the initial data points are not normally distributed and could suggest a wrong data sample compared to the population. 
```

Comment:
Does the last two sentence add anything to our point? It only raises the question why it is not the case with our data... Maybe it is more important to mention that normality is not requered for our model.